xivo_dao/resources/utils/search.py

```diff
@@ -194,3 +175,29 @@ def _paginate(self, query, limit=None, offset=0):
             query = query.limit(limit)
 
         return query
+
+    def _extract_search_params(self, parameters):
+        parameters = self._populate_parameters(parameters)
+        self._validate_parameters(parameters)
+        self.config.column_for_sorting(parameters['order'])
+
+        order = parameters.pop('order', None)
+        limit = parameters.pop('limit', None)
+        offset = parameters.pop('offset', 0)
+        reverse = False if parameters.pop('direction', 'asc') == 'asc' else True
+        return parameters, order, limit, offset, reverse
+
+    def _apply_search_params(self, rows, order, limit, offset, reverse):
+        if order:
+            rows = sorted(
+                rows,
+                key=lambda x: unicodedata.normalize('NFKD', x.__getattribute__(order)),
```

Comment:
```suggestion
                key=lambda x: unicodedata.normalize('NFKD', getattr(x, "order")),
```
getattr is the public interface for dynamic attribute access.